src/notify.ts

```diff
@@ -101,7 +102,9 @@ const sendNotification = async (
     try {
       const cids = JSON.parse(chat.contactIds || '[]')
       const notme = cids.find((id) => id !== 1)
-      const other = models.Contact.findOne({ where: { id: notme } })
+      const other: Contact = models.Contact.findOne({
```

Comment:
is this correct should we not await this command? cause also other.blocked wouldn't be an attribute on a promise would it?